src/lib/Microsoft.Health.Events/Telemetry/Metrics/EventMetrics.cs

```diff
@@ -100,5 +103,24 @@ public static Metric EventWatermark(string partitionId, DateTime dateTime)
                     { _categoryDimension, Category.Latency },
                 });
         }
+
+        /// <summary>
+        /// A metric recorded when there is an error reading from or connecting with an Event Hub.
+        /// </summary>
+        /// <param name="exceptionName">The name of the exception</param>
+        /// <param name="connectorStage">The stage of the connector</param>
+        public static Metric HandledException(string exceptionName, string connectorStage)
+        {
+            return new Metric(
+                exceptionName,
+                new Dictionary<string, object>
+                {
+                    { _nameDimension, exceptionName },
+                    { _categoryDimension, Category.Errors },
+                    { _errorTypeDimension, ErrorType.GeneralError },
+                    { _errorSeverityDimension, ErrorSeverity.Critical },
```

Comment:
Should all of the EventHubsExceptions be critical?

Reply:
I think we should default to critical for now. I have only been able to generate these types of errors when there is some major connectivity or config issue that would prevent the connecting from fully functioning. 

If there was some non-critical exception that was thrown then we can handle it as a non-critical error, but so far have not seen one that is non-critical.

Reply:
Sounds good. We can always make updates as needed.